verification_tools/gather_saturation_limits.py

```diff
@@ -129,22 +129,21 @@
 
 # http://bokeh.pydata.org/en/1.1.0/docs/user_guide/interaction/callbacks.html#userguide-interaction-jscallbacks
 scode = """
-        var active = cb_obj.active;
-        var labels = cb_obj.labels;
         var active_labels = [];
-        for (i = 0; i < active.length; i++) {
-            active_labels.push(labels[active[i]].toLowerCase());
+        for (let i = 0; i < this.active.length; i++) {
```

Comment:
I'm probably violating some JS code quality principle by mixing 'let' and 'var'.